my_2d_robot/src/my_2d_robo_state_publisher.cpp

```diff
@@ -0,0 +1,47 @@
+#include <string>
+#include <ros/ros.h>
+#include <sensor_msgs/JointState.h>
+#include "std_msgs/String.h"
+
+
+int main(int argc, char** argv){
+    ros::init(argc, argv, "my_2d_robo_state_publisher");
+    ros::NodeHandle n;
+    ros::Publisher joint_pub = n.advertise<sensor_msgs::JointState>("joint_states", 1000);
+    ros::Rate loop_rate(30);
+
+
+    sensor_msgs::JointState joint_state;
+    
+
+    
+    while(ros::ok){
+        joint_state.header.stamp = ros::Time::now();
+        //one message per update, This is 2 since thre are 2 joints which are updated 
```

Comment:
Form all comments as proper sentences, starting with capital letters and full stops at the end.